install-r-packages.R

```diff
@@ -1,39 +1,73 @@
+# This script attempts to install all the R dependencies for the narrative base image.
+# It goes through all the packages listed, attempts to install them, and saves an error
+# message if the install fails.
```

Comment:
Rejigged the install script a bit (thanks chatGPT!) - it tries to install everything and collects errors as it goes along. If there are failures, the error messages are listed and the install script exits with a non-zero status.